mlflow/models/evaluation/default_evaluator.py

```diff
@@ -248,49 +290,61 @@ def _gen_classifier_curve(
     :param y_probs: if binary classifier, the predicted probability for positive class.
                     if multiclass classifier, the predicted probabilities for all classes.
     :param labels: The set of labels.
+    :param pos_label: The positive label to use when computing classification metrics such as
+                      precision, recall, f1, etc. for binary classification models (default: ``1``).
+                      For multiclass classification and regression models, this parameter will be
+                      ignored.
     :param curve_type: "pr" or "roc"
+    :param sample_weights: Optional sample weights.
     :return: An instance of "_Curve" which includes attributes "plot_fn", "plot_fn_args", "auc".
     """
     if curve_type == "roc":
 
-        def gen_line_x_y_label_fn(_y, _y_prob):
-            fpr, tpr, _ = sk_metrics.roc_curve(_y, _y_prob)
-            auc = sk_metrics.auc(fpr, tpr)
-            return fpr, tpr, f"AUC={auc:.3f}"
+        def gen_line_x_y_label_auc(_y, _y_prob, _pos_label):  # pylint: disable=unused-argument
+            fpr, tpr, _ = sk_metrics.roc_curve(_y, _y_prob, sample_weight=sample_weights)
+            auc = sk_metrics.roc_auc_score(y_true=_y, y_score=_y_prob, sample_weight=sample_weights)
```

Comment:
```suggestion
            auc = sk_metrics.roc_auc_score(y_true=_y, y_score=_y_prob, sample_weight=sample_weights, pos_label=pos_label)
```

Do we need to pass `pos_label` here? Related to https://github.com/mlflow/mlflow/pull/6802#discussion_r971522950.

Reply:
Never mind.

Reply:
Correct :)

Reply:
Wait, we have `roc_curve` at line 304. Should we use `pos_label` there?

Reply:
+1. We should put `pos_label` in L304 as well.